src/documentation/0040a-node-request-data/index.md

```diff
@@ -68,7 +68,7 @@ const server = http.createServer((req, res) => {
     data.push(chunk)
   })
   req.on('end', () => {
-    JSON.parse(data).todo // 'Buy the milk'
+    JSON.parse(data.toString()).todo // 'Buy the milk'
```

Comment:
Wouldn't this [insert commas](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/toString#Description) that were not present in the original chunks pushed into the array?

Reply:
the porblem is JSON.parse only accepts [a one string  argument](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/parse) however now i can see that my fix is also flawed , a simple way to fix this will be just to print the array as its without transforming it into an object